server/src/test/java/io/crate/analyze/SubSelectAnalyzerTest.java

```diff
@@ -316,4 +317,14 @@ public void test_exists_is_printed_as_exists_keyword() {
         QueriedSelectRelation relation = analyze("select * from sys.summits where EXISTS (select 1)");
         assertThat(relation.where()).hasToString("EXISTS (SELECT 1 FROM (empty_row))");
     }
+
+    @Test
+    public void test_outputs_from_nested_aliased_analyzed_relation_contain_all_relation_names_to_be_distinct() {
+        QueriedSelectRelation relation = analyze("select a from (select a from (select a from t1 as tbl1) tbl2) tbl3");
+        Symbol symbol = relation.outputs().get(0);
+        assertThat(symbol)
+            .isField("a", new RelationName(null, "tbl3"))
+            .isField("a", new RelationName(null, "tbl2"))
+            .isField("a", new RelationName(null, "tbl1"));
```

Comment:
For me, chaining the `isField` commands is confusing and decreases readability. I could not understand that a chained `isField` will assert on the `ScopedSymbol.symbol()`.
```suggestion
        assertThat(symbol).isField("a", new RelationName(null, "tbl3"));
        assertThat(symbol.symbol()).isField("a", new RelationName(null, "tbl2"))
        assertThat(symbol.symbol().symbol()).isField("a", new RelationName(null, "tbl1"));
```